### PR TITLE
Update netlify publish settings

### PR DESCRIPTION
**Pull request by theinterned**
resolves [NDS-118](https://nulogy-go.atlassian.net/browse/NDS-118)

This pull requests fixes the http://nulogy.design Netlify continuous deploy config to work with the new monorepo structure.

At the moment, this branch is set as the "production" branch in Netlify. When this PR is accepted I will update that setting to use the "master" branch again.

# What I did:

1. Added a yarn build script and updated the netlify.toml to use that script.
1. Renamed the directory that the website package lives in from `style-guide/` to `docs/`
1. Updated some sections of the docs (please copy edit these carefully as I am dyslexic and a horrible typo-ist): 
   - Added content for http://nulogy.design/getting_started/setup
   - Added content for http://nulogy.design/getting_started/deploying
   - Removed the `/getting_started/designer` page: We decided we can bring this back if we find a need for it.
1. In order to get the build to work, I had to specify a [hard-coded yarn version in the ./netlify.toml`](https://github.com/nulogy/design-system/blob/2775f413c14bc700adccaab3f59d6e3447dbdd71/netlify.toml#L12). We'll have to try to keep tabs on this and remove it when Netlify upgrades their default version of yarn.

# To test: 

## Build configuraion

As the deploy settings are currently locked in my private account, you can't verify that the deploy is working. You can check out the configuration changes I've made. You can also verify that the content updates I made are live on the site now: http://nulogy.design/getting_started/setup

## Doc updates

Please have a look at the following added docs. I'm looking for copy editing as well as comments on ease-of understanding and style.

### [Getting Started / Set up](http://nulogy.design/getting_started/setup)
 
https://github.com/nulogy/design-system/blob/2775f413c14bc700adccaab3f59d6e3447dbdd71/docs/catalog/getting_started/setup.md

### [Getting Started / Deploying to the web](http://nulogy.design/getting_started/deploying)

https://github.com/nulogy/design-system/blob/2775f413c14bc700adccaab3f59d6e3447dbdd71/docs/catalog/getting_started/deploying.md

### CODE REVIEW

File: docs/catalog/getting_started/setup.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+---
+$ yarn -v
+bash: command not found: yarn 
+```
+
+ ... then you'll need to install yarn by following the instructions for your platform on the yarn website: [https://yarnpkg.com/lang/en/docs/install](https://yarnpkg.com/lang/en/docs/install)
+
+## Install dependencies and link local packages
+
+Yarn workspaces allows you to install dependencies for all packages in the monorepo in one command.
```
Comment (matthewlyle) [inline docs/catalog/getting_started/setup.md]: typo, extra s. "Yarn workspaces allow you..."

**Review by matthewlyle — COMMENTED** [review]
Setup instructions worked well! Got the docs running locally without interruption.

File: docs/catalog/getting_started/deploying.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  1. Enter the url provided in the terminal as the "Payload URL"
+  1. Select "`application/json`" as the "Content type".
+  1. Click the "Let me select individual events" radio button.
+    1. Choose "Pushes" and "Pull requests".
+
+If all goes well you will be given a Netlify url. Now whenever you push or merge to master on your fork on GitHub, it will trigger a build.
+
+## Netlify account
+
+At the moment, the Netlify account is run by [Ned Schwartz](https://github.com/nulogy). Get in touch if you need help!
```
Comment (matthewlyle) [inline docs/catalog/getting_started/deploying.md]: This is just a link to the Nulogy Github, not how to contact you. Maybe it should be your email address? Or just a note to reach out on Slack instead?

**Comment by theinterned** [comment]
From our dry run in office we found a few missing pieces in the setup instructions:

- [x] Add instructions for installing NVM (including possibly creating a `.profile` if installation fails)
- [x] Add direnv installation https://github.com/direnv/direnv
- [x] Add step to build the components before running the docs
- [x] Simplify the instructions for running commands in the project (currently there are 3 ways to do things - make this simpler)

File: docs/catalog/getting_started/deploying.md (hunk 2 above)
Comment (theinterned) [reply docs/catalog/getting_started/deploying.md]: Whoops - copy+paste error: this now points to my github profile

File: docs/catalog/getting_started/setup.md (hunk 1 above)
Comment (theinterned) [reply docs/catalog/getting_started/setup.md]: I think in this case "yarn workspaces" is a collective noun and should be treated as singular.

Comment (theinterned) [reply docs/catalog/getting_started/setup.md]: My editor wife suggests capitalizing the W

Comment (theinterned) [reply docs/catalog/getting_started/setup.md]: Looks like Yarn can't even decide if its singular or plural:

> Workspaces **are** a new way to setup your package architecture that’s available by default starting from Yarn 1.0. **It allows** you to setup multiple packages in such a way that you only need to run yarn install once to install all of them in a single pass.

Comment (theinterned) [reply docs/catalog/getting_started/setup.md]: I rewrote the sentence to add a more stable subject

**Comment by theinterned** [comment]
@matthewlyle I've made updates based on your feedback. Have a look!

**Review by matthewlyle — APPROVED** [review]

**Pull request merged**